java.js

```diff
@@ -0,0 +1,14 @@
+const myopen = document.querySelector('open');
+const nav = document.querySelector('.nav-links-2');
+const myaxe = document.querySelector('.axe');
+const mytra1 = document.querySelector('.nav-transition-1');
+
+myopen.onclick = function () {
+  nav.style.cssText = 'transform: translateY(0%);';
+};
+myaxe.onclick = function () {
+  nav.style.cssText = 'transform: translateY(-100%);';
+};
+mytra1.onclick = function () {
+  nav.style.cssText = 'transform: translateY(-100%);';
+};
```

Comment:
```
In mobile, implement the following features:

When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
```

- Currently on click none of the functionalities above is implemented, please make sure that the click events above are implemented see the video below ⬇️ 

https://user-images.githubusercontent.com/69638013/230952092-1d3dccf3-d345-412a-b563-5a6a6a37dd81.mp4